package.nls.de.json

```diff
@@ -0,0 +1,27 @@
+{
+  "command.regenerateIcons.title": "Icon-Veränderungen anwenden",
+  "command.ngPreset.title": "Angular Preset umschalten (Workspace Level)",
+  "command.jsPreset.title": "Offizielles JS Preset umschalten (User Level)",
+  "command.tsPreset.title": "Offizielles TS Preset umschalten (User Level)",
+  "command.jsonPreset.title": "Offizielles JSON Preset umschalten (User Level)",
+  "command.hideFoldersPreset.title": "Ordner-Icons Sichtbarkeit umschalten (User Level)",
+  "command.foldersAllDefaultIconPreset.title": "Spezifische Order-Icons umschalten (User Level)",
+  "command.restoreIcons.title": "Icon-Manifest zurücksetzen",
+  "command.resetProjectDetectionDefaults.title": "Projekterkennung zurücksetzen",
+  "configuration.title": "vscode-icons Einstellungen",
```

Comment:
`vscode` itself uses the word `Konfiguration` instead of `Einstellungen`. `Einstellungen` means `Settings`.